runing/config.py

```
@@ -0,0 +1,12 @@
+from pydantic_settings import BaseSettings
+
+class Settings(BaseSettings):
+    # TODO: YOUR_PASSWORD 부분을 실제 postgres 사용자 비밀번호로 변경하세요.
```

Pylint (3.3.7)

[warning] TODO: YOUR_PASSWORD 부분을 실제 postgres 사용자 비밀번호로 변경하세요.

(W0511)